hindi/art-of-living/art-of-living.md

```diff
@@ -0,0 +1,3 @@
+Founded in 1981 by Sri Sri Ravi Shankar,The Art of Living is an educational and humanitarian movement engaged in stress-management and service initiatives. The organization operates globally in 154 countries and has touched the lives of over 370 million people.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.